static/templates/recent_topics_filters.hbs

```diff
@@ -23,3 +23,6 @@
     {{/if}}
     {{t 'Participated' }}
 </button>
```

Comment:
Move the stream filter widget before input box so that the change in the width of the button when selected stream is changed will be consumed by the input box and not move other filters around.